### DISCUSSION TITLE
Clean up the useless "select station" drop down menu for rides that have only 1 station

### DISCUSSION THREAD

**Discussion by luotiansha** [OpenRCT2/OpenRCT2#21871] [Feature requests]
Wouldn't it be better if this drop down menu is just left with only an "overall view" option, since these rides can only have 1 station anyway? Right now there are many non-railway rides that have this useless menu.

<img width="592" alt="截屏2024-04-22 04 21 19" src="https://github.com/OpenRCT2/OpenRCT2/assets/77926812/23d6cdb7-5d58-41da-8659-6cc05e02c69d">

**Comment by MASHzj** [comment]
I often use both the overall view and the station. The former shows the total of guests on your ride, the latter shows the number of guests in the queue.

**Reply by astraylife1** [reply]
Same, I use the station view to see how many guests are in queue

**Reply by luotiansha** [reply]
Oh I see, it _is_ useful then, I didn’t notice that before.

**Comment by 733737** [comment]
Overall view shows a zoomed out view with surrounding rides, station view allows for the station audio option
